presto-hive/src/test/java/com/facebook/presto/hive/TestCteExecution.java

```diff
@@ -1261,9 +1257,24 @@ protected Session getSession()
     protected Session getMaterializedSession()
     {
         return Session.builder(super.getSession())
+                .setSystemProperty(VERBOSE_OPTIMIZER_INFO_ENABLED, "true")
                 .setSystemProperty(PUSHDOWN_SUBFIELDS_ENABLED, "true")
                 .setSystemProperty(CTE_MATERIALIZATION_STRATEGY, "ALL")
                 .setSystemProperty(CTE_FILTER_AND_PROJECTION_PUSHDOWN_ENABLED, "true")
                 .build();
     }
+
+    public static class CTEInfoValue
```

Comment:
Also can we just reuse the CTEInformation class from the session?

Reply:
Good point. It didn't register in my mind to use the CTEInformation class. I can created a CTEInformation rather than a CTEInfoValue, but I won't have access to a cteId value so it will stay null or match the name. Is that acceptable - otherwise I will just continue to use the CTEInfoValue inner class and extend it with the isMaterialized boolean? As mentioned in the earlier comment, it would have been great to just use the CTEInformation from the Session object directly, but that Session is created in-line when the DispatchManager is creating an internal query which the tests doesn't have access to.

Reply:
> I won't have access to a cteId value so it will stay null or match the name.
Yes same is fine.